gremlinapi/http_clients.py

```diff
@@ -60,48 +67,45 @@ def header(cls, *args, **kwargs):
             else:
                 header["Authorization"] = f"Bearer {bearer_token}"
         else:
-            error_msg = f"Missing API Key or Bearer Token, none supplied: {api_key}, {bearer_token}"
-            log.fatal(error_msg)
+            error_msg: str = f"Missing API Key or Bearer Token, none supplied: {api_key}, {bearer_token}"
+            log.error(error_msg)
             # raise HTTPBadHeader(error_msg)
         header["X-Gremlin-Agent"] = f"gremlin-sdk-python/{get_version()}"
         return header
 
     @classmethod
-    def proxies(cls):
-        error_message = f"This function is not implemented, please consume the proper http library for your environment"
-        log.fatal(error_message)
+    def proxies(cls) -> dict:
+        error_message: str = f"This function is not implemented, please consume the proper http library for your environment"
+        log.error(error_message)
         raise NotImplementedError(error_message)
 
 
-class GremlineAPIRequestsClient(GremlinAPIHttpClient):
+class GremlinAPIRequestsClient(GremlinAPIHttpClient):
```

Comment:
Refactor to replace Union[object, object] in all files that consume these classes